digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/process/TrackSectionOrder.scala

```diff
@@ -79,6 +79,36 @@ object TrackSectionOrder {
     }
   }
 
+  def groupConnectionPoints(seq: Seq[ProjectLink]): Map[(Double, Double), Seq[Point]] = {
```

Comment:
/** Group together points of the given ProjectLinks having the same (x,y) coordinates. */

Reply:
Jes, nyt on sen kokoisia funkkareita, että näistä pystyy saamaan tolkun! \o/